act/policy.go

```diff
@@ -9,7 +9,7 @@ import (
 
 type IPolicy interface {
 	MustCompile(opts ...expr.Option) error
-	Eval(ctx context.Context, input Input) (any, error)
+	Eval(ctx context.Context, input any) (any, error)
```

Comment:
Why has this become `any`?

Reply:
oh, I was trying something while testing the code, and missed to revert it back. it is fixed now